### DISCUSSION TITLE
Support for Raspberry Pi Zero 2 W?

### DISCUSSION THREAD

**Discussion by odymaui** [PiSupply/PiJuice#889] [Q&A]
I have seen confirmation that the Raspberry Pi Zero W is supported with PiJuice Hat but is the Raspberry Pi Zero 2 W supported?  The models should be basically the same but just want to confirm since I can't find any reference to explicitly confirm Raspberry Pi Zero 2 W support.

**Comment by tvoverbeek** [comment]
Why shouldn't it be? GPIO interface (PiJuice ony uses I2C and the power and ground pins) is exactly the same.
THe same holds for the original 40 pin Raspberry Pis without WiFi (B+, 2B) and the later ones with WiFi (3B, 3B+, 4B)
